src/ReinforcementLearningFarm/test/runtests.jl

```diff
@@ -0,0 +1,17 @@
+using UUIDs
+using Preferences
+
+if Sys.isapple()
+    flux_uuid = UUID("587475ba-b771-5e3f-ad9e-33799f191a9c")
+    set_preferences!(flux_uuid, "gpu_backend" => "Metal")
+
+    using Metal
+else
+    using CUDA, cuDNN
+    CUDA.allowscalar(false)
+end
```

Comment:
Not a priority but there's also an AMD gpu support package now. I don't know if it is mature enough.